### PR TITLE
Update simulator configuration

### PR DESCRIPTION
**Pull request by katiemacaulay**
- updated input configuration
- tags need to be updated
- should client_settings be omitted from the input configuration?

### CODE REVIEW

File: docs/monoDrive_home/Simulator-Configuration.md [hunk 1]
```diff
@@ -2,135 +2,132 @@
...
-         "specular_exponent":80.0,
-         "specular_coefficient":0.80,
-         "diffuse_coefficient":0.15,
-         "dielectric_constant":4.0,
-         "roughness":0.10
+      "vehicle_dynamics": "physx"
+   },
+   "id": "simulator_test",
+   "map": "Almono",
+   "phys_materials": {
```
Comment (bradleynull) [inline docs/monoDrive_home/Simulator-Configuration.md:38]: It looks like the default physical materials in the C++ client are not the latest values. Replace these guys with the ones from the LV client. Opened [this issue](https://github.com/monoDriveIO/monodrive-client/issues/44) to address the problem. Below should be the latest:

```json
{
    "phys_materials": {
    "Aluminum": {
      "specular_exponent": 15.0,
      "specular_coefficient": 0.95,
      "diffuse_coefficient": 0.26,
      "dielectric_constant": 10.0,
      "roughness": 0.15
    },
    "Asphalt": {
      "specular_exponent": 1.0,
      "specular_coefficient": 0.03,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.65
    },
    "Concrete": {
      "specular_exponent": 0.0,
      "specular_coefficient": 0.1,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.0
    },
    "Glass": {
        "specular_exponent": 80.0,
        "specular_coefficient": 0.80,
        "diffuse_coefficient": 0.15,
        "dielectric_constant": 4.0,
        "roughness": 0.10
      },
    "Gravel": {
        "specular_exponent": 10.0,
        "specular_coefficient": 0.1,
        "diffuse_coefficient": 0.95,
        "dielectric_constant": 7.0,
        "roughness": 0.48
      },
    "LandScape": {
        "specular_exponent": 10.0,
        "specular_coefficient": 0.50,
        "diffuse_coefficient": 0.50,
        "dielectric_constant": 10.0,
        "roughness": 0.60
      },
    "Plastic": {
        "specular_exponent": 25.0,
        "specular_coefficient": 0.40,
        "diffuse_coefficient": 0.60,
        "dielectric_constant": 2.8,
        "roughness": 0.09
      },
    "ReflectiveMaterial": {
      "specular_exponent": 10.0,
      "specular_coefficient": 0.5,
      "diffuse_coefficient": 0.5,
      "dielectric_constant": 1.0,
      "roughness": 1.0
    },
    "Road": {
      "specular_exponent": 1.0,
      "specular_coefficient": 0.1,
      "diffuse_coefficient": 0.65,
      "dielectric_constant": 7.0,
      "roughness": 0.65
    },
    "RoadPaint": {
        "specular_exponent": 10.0,
        "specular_coefficient": 10.0,
        "diffuse_coefficient": 2.0,
        "dielectric_constant": 7.0,
        "roughness": 1.0
      },
    "Steel": {
      "specular_exponent": 26.0,
      "specular_coefficient": 1.0,
      "diffuse_coefficient": 0.33,
      "dielectric_constant": 0.0,
      "roughness": 0.23
    },
    "Tire": {
      "specular_exponent": 25.0,
      "specular_coefficient": 0.40,
      "diffuse_coefficient": 0.60,
      "dielectric_constant": 2.8,
      "roughness": 0.15
    },
    "Tree": {
      "specular_exponent": 10.0,
      "specular_coefficient": 0.05,
      "diffuse_coefficient": 0.97,
      "dielectric_constant": 10.0,
      "roughness": 0.60
    },
    "Wheel": {
      "specular_exponent": 92.0,
      "specular_coefficient": 0.95,
      "diffuse_coefficient": 0.06,
      "dielectric_constant": 10.0,
      "roughness": 0.15
    }
  }
```

File: docs/monoDrive_home/Simulator-Configuration.md [hunk 2]
```diff
@@ -2,135 +2,132 @@
...
+      "spawn_rate": 0.25
    }
 }
 ```
 <p>&nbsp;</p>
 
 ### Configuration Tags
 
-- **server_ip**: The ip address of the computer running the simulator. If both simulator and client are running on the same computer use localhost as the ip address (127.0.0.1).
+- **client_ip**: The ip address of the computer running the simulator. If both simulator and client are running on the same computer use localhost as the ip address (127.0.0.1).
```
Comment (bradleynull) [inline docs/monoDrive_home/Simulator-Configuration.md:130]: This is actually an addition to the configuration. For some reason the `server_ip` was dropped in the config file I sent to you. Can you add `server_ip` back into the mix and use this description for client and server:

```
**client_ip**: The IP address of the computer running the client software. 
**server_ip**: The IP address of the computer running the monoDrive simulator.
```

File: docs/monoDrive_home/Simulator-Configuration.md [hunk 3]
```diff
@@ -2,148 +2,173 @@
  
 ## Simulator Configuration
 
-```
+``` json
```
Comment (citlaligm) [inline docs/monoDrive_home/Simulator-Configuration.md:5]: We lost the `"traffic_configuration":` section this is used. Same for the tag descriptions we need to put them back

File: docs/monoDrive_home/Simulator-Configuration.md [hunk 4]
```diff
@@ -2,148 +2,173 @@
...
-  - **max_desired_speed**: Specify the maximum amount the speed of any vehicle can be **increased** from the speed limit of the road. Units on km per hour.
-  - **swarm_traffic"**: Set to *True* if you want to control the distance where other vehicles can spawn as well as when they can disappear from the map.
-  - **spawn_leash_distance**: If **swarm_traffic** enabled. Specify the maximum distance a car can be from the ego vehicle before it gets destroyed. Units on centimeters.
-  - **min_spawn_range**: If **swarm_traffic** enabled. Specify the minimum distance where a car can be spawn from the ego vehicle. Units on centimeters.
-  - **vertical_offset**: Specify the distance from the road a car is being spawn on the Z direction.
-  - **overtake_slower_vehicles**: Set to *True* if you want to allow cars at a faster speed to overtake cars moving slower. 
-- **phys_materials**: Specify the physical material properties for the specular exponent (10-100), specular reflection coefficient (0.0-1.0), diffuse reflection coefficient (0.0-1.0), dielectric constant (0.10-10,000+), and roughness (0.0-1.0).
+  - **spawn_rate**:
```
Comment (citlaligm) [inline docs/monoDrive_home/Simulator-Configuration.md:172]: We can remove this one.

File: docs/monoDrive_home/Simulator-Configuration.md [hunk 5]
```diff
@@ -2,148 +2,173 @@
...
+   - **gui**
+   - **logger**
+   - **map**
+      - **gis_anchor**
+      - **point_delta**
+- **ego_config**: Specifies the model and color of the ego vehicle. The `body`'s `type` specifier allows you to provide a path to the model BluePrint for the ego vehicle and the `color` specifies the vehicle's color. If `ego_config` is omitted from the file, then a random vehicle and color will be chosen. Only applies when the simulator is running on replay mode. For closed loop it will use the car specified in the scenario file.
+
+<!-- should mention something about vehicle dynamics- physx -->
+
+- **id**
```
Comment (citlaligm) [inline docs/monoDrive_home/Simulator-Configuration.md:162]: This is a name for the simulator configuration i.e. "simulator with no traffic"

**Pull request merged**